scene/main/node.h

```diff
@@ -86,6 +86,12 @@ class Node : public Object {
 		FLAG_PROCESS_THREAD_MESSAGES_ALL = 3,
 	};
 
+	enum PhysicsInterpolationMode {
```

Comment:
```
	enum PhysicsInterpolationMode : unsigned int {
```
When using enums in bitfields, always specify unsigned. Some compilers default to signed for enums, and this can overflow the bits.

See:
https://github.com/godotengine/godot/pull/61757